term-utils/script.c

```diff
@@ -408,6 +409,9 @@ static int log_start(struct script_control *ctl,
 			fprintf(log->fp, _("<not executed on terminal>"));
 
 		fputs("]\n", log->fp);
+
+		fprintf(log->fp, _("%s"), ctl->command);
+		fputs("\n", log->fp);
```

Comment:
If the translation is not needed, you can merge the two `fprintf` call:

```
if (ctl->command)
    fprintf(log->fp, "%s\n", ctl->command);
```

Reply:
This will definitely take the NULL case into consideration.

Reply:
Implemented in PR